<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonBuilderWizard</class>
 <widget class="QWizard" name="PersonBuilderWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New person</string>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <property name="title">
    <string>Person</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="firstNameLabel">
      <property name="text">
       <string>First name:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="firstNameLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lastNameLabel">
      <property name="text">
       <string>Last name:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lastNameLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="genderLabel">
      <property name="text">
       <string>Gender:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="genderGroupBox">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="feminineGenderPushButton">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="masculineGenderPushButton">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <property name="title">
    <string>Birth</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="dateLabel">
      <property name="text">
       <string>Date:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="countryLabel">
      <property name="text">
       <string>Country:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="countryLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="departmentLabel">
      <property name="text">
       <string>Department:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="departmentLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="cityLabel">
      <property name="text">
       <string>City:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="cityLineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
